tests/python-sycl/test_sycl_prediction.py

```diff
@@ -0,0 +1,155 @@
+import sys
```

Comment:
Consider using the `black` formatter for new files: https://github.com/dmlc/xgboost/blob/fedd9674c8441a26b94f9d4061b11a1be4cb0e7c/tests/ci_build/lint_python.py#L16

Reply:
done